Resources/Styles/package_list.scss

```diff
@@ -34,7 +34,15 @@
       p {
         margin: 5px 0;
         color: var(--page-text);
+        
+        &.stars {
```

Comment:
@daveverwer can you take a look at the styling on this? It's never taking effect or getting compiled into the `main.css` when I run everything locally and I'm confused about what could be causing that.

Reply:
<img width="915" alt="Screen Shot 2021-08-11 at 8 57 25 PM" src="https://user-images.githubusercontent.com/9971367/129126807-0a53deb5-9dfa-46f9-8b52-de1ca6e44711.png">
^^That is what I'm ending up with locally.

Reply:
It's possible you're not running `make serve-front-end` as you develop.

The CSS and JavaScript only get compiled from the SASS and multi-file JS sources when Rollup processes them and minifies them.

`make build-front-end` does a one-time build into the correct directory and is what we run during deployment to make sure that the live CSS is always correct. `make serve-front-end` is what you should run locally, and it also watches the source files for changes and re-compiles whenever it needs to.

If it's not that, let me know and we can investigate more!

Reply:
That's definitely it, but I didn't realize it as I was getting a styled site with just a `make run`. I'm pretty sure I ran `make serve front-end` on my first pr, so then I must have had a `main.css` hanging around from that somehow...

Reply:
> I must have had a `main.css` hanging around from that somehow...

Yep.

It's not ideal, but it's the best we have come up with so far. The compiled `main.css`/`main.js` and associated files are not checked into source code, which is beneficial so you only need to commit the `.scss` files and source JavaScript files, not the minified ones.

The downside is that you can have an old version of those files hanging around for weeks without noticing it as it's not tracked by git.

Build phases for `Package.swift` files would save us here, but they're not quite here yet.